release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -2653,6 +2653,8 @@ and recreate it without a BFD profile. (link:https://bugzilla.redhat.com/show_bu
 
 * If a cluster that was deployed through ZTP has policies that do not become compliant, and no `ClusterGroupUpdates` object is present, you must restart the {cgu-operator} pods. Restarting {cgu-operator} creates the proper `ClusterGroupUpdates` object, which enforces the policy compliance. (link:https://issues.redhat.com/browse/OCPBUGS-4065[*OCPBUGS-4065*])
 
+* Currently, when using a persistent volume (PV) that contains a very high file count, the pod may not start or takes an excessive amount of time to start. For more information, see this link:https://access.redhat.com/solutions/6221251[knowledge base article]. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1987112[*BZ1987112*])
```

Comment:
```suggestion
* Currently, when using a persistent volume (PV) that contains a very high file count, the pod may not start or may take an excessive amount of time to start. For more information, see this link:https://access.redhat.com/solutions/6221251[knowledge base article]. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1987112[*BZ1987112*])
```

Reply:
I guess the reason is the title is high file counts in https://access.redhat.com/solutions/6221251